tutorials/configure-dvc-with-object-storage/index.mdx

```diff
@@ -0,0 +1,144 @@
+---
+meta:
+  title: Configuring DVC with Object Storage
+  description: This page provides information on how to configure DVC with Scaleway Object Storage.
+content:
+  h1: Configuring DVC with Object Storage
+  paragraph: This page provides information on how to configure DVC with Scaleway Object Storage.
+tags: object-storage storage amazon-s3 data-version-control s3-compatible dvc-bucket machine-learning data-science
+categories:
+  - storage
+  - object storage
+dates:
+  validation: 
+  posted: 
+---
+
+## Data Version Control (DVC) - Overview
+
+Git is unarguably the most popular and powerful version control system to store your code, and can handle files up to 5 GB thanks to Git LFS. But what about large datasets?
+
+In that case, you might need to turn to third-party version control tools that are designed to handle large datasets. 
+
+Data Version Control (DVC) was specifically designed with this use-case in mind. It works alongside Git and allows you to store your data in the remote storage of your choice (such as a Scaleway S3-enabled bucket) while storing only the metadata in a Git repository.
+
+In this tutorial, you learn how to use [Scaleway Object Storage](https://www.scaleway.com/en/object-storage/) as a remote storage for DVC.
+
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created a bucket](/storage/object/how-to/create-a-bucket/)
+  - You have [created an API key](https://www.scaleway.com/en/docs/identity-and-access-management/iam/how-to/create-api-keys/)
+  - You have created a repository to store your metadata.
+  - You have made your [first request with Scaleway API](https://www.scaleway.com/en/developers/api/#quickstart:-first-request)
+  - You [authenticated to the API](https://www.scaleway.com/en/developers/api/#how-to-authenticate) for the first time 
+  - You have installed the [AWS CLI](/storage/object/api-cli/object-storage-aws-cli/)
+</Message> 
+
+## Setting up DVC
+
+1. Run the following command to install the DVC Python package:
+    ```bash
+    pip3 install dvc
+    ```
+
+2. Run the following command to install the S3 dependencies:
+    ```bash
+    pip3 install "dvc[s3]"
+    ```
+
+3. Run the following command in the desired repository to initialize DVC:
+    ```bash
+    dvc init
+    ```
+
+4. Run the following command to commit the initial DVC configuration files:
+    ```bash
+    git commit -m "Initialize DVC"
+    ```
+
+## Retrieving and tracking data
+
+1. Run the following command to copy the example data file to your repository:
+    ```bash
+    dvc get https://github.com/iterative/dataset-registry \
```

Comment:
Does not work: https://github.com/iterative/dataset-registry/get-started/data.xml

Reply:
I just re-run the command, it works, I can't reproduce.